store/src/java/com/zimbra/util/ExternalVolumeInfoHandler.java

```diff
@@ -192,4 +192,26 @@ public void modifyServerProperties(VolumeInfo volInfo) throws ServiceException,
             throw e;
         }
     }
+
+    /**
+     * Validate global bucket ID by reading LDAP atrribute
+     * @param globalS3BucketId
+     * @returns true if "global bucket ID" is valid
+     * @throws ServiceException, JSONException
+     */
+    public Boolean validateGlobalBucketID(String globalS3BucketId) throws ServiceException, JSONException {
+        // step 1: Fetch globalS3Config JSON state object and globalS3Config JSON state array
+        String globalExternalStoreConfigJSON = provisioning.getConfig().getGlobalExternalStoreConfig();
```

Comment:
suggestion for var naming
`globalExternalStoreConfigJSON` => `globalExternalStoreConfig`
`globalS3ConfigJSONObject` => `globalS3Configs`
`globalS3ConfigJSONArray` => `globalS3ConfigList`

Reply:
Fixed